### DISCUSSION TITLE
Octicon Icons is displayed upside down in the footer

### DISCUSSION THREAD

**Discussion by fish-train** [pradyunsg/furo#484] [Potential Issue] [answered]
### What's happening?
An Octicon icon is displayed upside down for the Previous button in the footer
![image](https://user-images.githubusercontent.com/59873039/179945356-c02b569d-445a-4ef2-a970-776120da1b95.png)

### Reproducer

1. Set the **Sphinx Design** extension.
1. Add An Octicon icon in the header. For example, `:octicon:`rocket;1em` Run a test`. For more information, see [https://sphinx-design.readthedocs.io/en/furo-theme/badges_buttons.html#octicon-icons](https://sphinx-design.readthedocs.io/en/furo-theme/badges_buttons.html#octicon-icons)
2. Build the Sphinx documentation set with Furo.
3. Visit the page on the built site, which is located after **Run a test**.
4. Scroll to the footer.

### Expectation
An Octicon icon is displayed correct.

**Comment by pradyunsg** [comment, marked as answer]
Indeed, and nice catch. Fixed in https://github.com/pradyunsg/furo/commit/8eba6499b812d7aeab2a99fcdf33e8bcb07b05fc

The problem was that Furo's stylesheet used `.related-pages a.prev-page svg` to rotate the svg it uses for the arrow, and that also matches what Sphinx design embeds in there.

**Reply by fish-train** [reply]
Thank you!
